starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,96 @@
+/*Import*/
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:300,600);
+
+/*Some necessary adjustments*/
+h1, body, html, main {
+  margin: 0;
+}
+/*Import the font and set the default size*/
+html {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px;
+}
+
+/*horizontal line separators*/
+hr {
+  color: darkgray;
+  width: 60%;
+}
+
+/*Header with background image*/
+header {
+  background: url("../images/apple-pie.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  height: 450px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  font-size: 1.5em;
+}
+
+/*General recipe info*/
+#general-info {
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+}
+
+#general-info p {
+  width: 60%;
+  text-align: center;
+}
+
+#general-info img {
+  display: block;
+  width: 25%;
```

Comment:
without using flex in the parent you could have added `margin: 0 auto` to center this image (because you added the display: block, otherwise it wouldn't work)